Sources/xcodeproj/Utils/Decoders.swift

```diff
@@ -28,10 +33,12 @@ class ProjectDecodingContext {
 
     /// Objects.
     let objects: PBXObjects
+    let jsonDictionary: [String: Any]
 
-    init() {
+    init(jsonDictionary: [String: Any] = [:]) {
```

Comment:
Do you need to have a default value here?

Reply:
Yes. This object is also widely used as a default parameter.